pkg/apis/kops/componentconfig.go

```diff
@@ -500,6 +500,9 @@ type KubeAPIServerConfig struct {
 	// in "iss" claim of issued tokens. This value is a string or URI.
 	ServiceAccountIssuer *string `json:"serviceAccountIssuer,omitempty" flag:"service-account-issuer"`
 
+	// AdditionalServiceAccountIssuers can contain additional service account token issuers.
+	AdditionalServiceAccountIssuers []string `json:"additionalServiceAccountIssuers,omitempty" flag:"service-account-issuer,repeat"`
```

Comment:
Maybe remove the `flag` and just append these after the flags are built, like here:
https://github.com/kubernetes/kops/blob/c6bb40ed328c99ca1490113334f61622600c5155/nodeup/pkg/model/kube_apiserver.go#L576
```suggestion
	AdditionalServiceAccountIssuers []string `json:"additionalServiceAccountIssuers,omitempty"`
```